pages/resources/counseling.js

```diff
@@ -3,16 +3,10 @@ import { supabase } from '../../utils/client';
 import { useRouter } from 'next/router';
 import Loader from '../../components/Loader';
 import MainLayout from '../../layouts/MainLayout';
-import { creative } from '../../data/resources';
 import { counselors } from '../../data/resources';
-import { creativeIconPics } from '../../data/resources';
 import Link from 'next/link'
-import { resourceTagData } from '../../data/tagData';
-import Tags from '../../components/Tags';
 
 
-const icons = creativeIconPics.map(icon => icon.url);
-
 export default function Couseling() {
```

Comment:
Looks like we have a misspelling here.

Reply:
Thank you! I'll edit and repush

Reply:
@bmgdevelopment make sure to pull down development into this branch as well. There seem to be some conflicts

Reply:
just seeing this, I will do a pull